addons/web_editor/static/lib/odoo-editor/README.md

```diff
@@ -0,0 +1,41 @@
+# Odoo editor
+
+A fast wysiwyg editor targeting document and website structure.
```

Comment:
"structures"

Reply:
then `documents`

Reply:
I thought you meant "document structure and website structure", then it would have been "document and website structures". But indeed it's "documents and website structures" :-)